js/functions.js

```diff
@@ -0,0 +1,51 @@
+// функция проверки длины строки
+
+const whatLength = (string, length) => {
+	if (string.length <= length) {
+		return true;
+	}
+	return false;
+};
+
+whatLength('Я путаюсь в JS', 16);
+
+// функция проверки на палиндром
+
+const isPalindrome = (string) => {
+	const normalizedString = string.replaceAll(' ', '').toLowerCase().trim();
+
+	let reversedString = '';
+
+	for (let i = normalizedString.length - 1; i >= 0; i--) {
+		reversedString += normalizedString[i];
+	}
+
+	if (normalizedString === reversedString) {
+		return true;
+	}
+
+	return false;
+};
+
+isPalindrome(' Лёша на полке клопа нашёл   ');
+
+// Функция принимает строку, извлекает содержащиеся в ней цифры от 0 до 9 и возвращает их в виде целого положительного числа
+
+const findDigits = (unknownString) => {
+	let foundNumbers = '';
```

Comment:
Не хватает прохождения на числа
![image](https://github.com/htmlacademy-javascript/2287579-kekstagram-29/assets/47776594/4217761e-73a8-44b7-897e-0bff07580808)

А решить легко

```suggestion
const findDigits = (unknownString) => {
	unknownString = unknownString.toString();
	let foundNumbers = '';
```

Reply:
спасибо!